apps/etl-func/src/adapters/config.ts

```diff
@@ -0,0 +1,56 @@
+import { pino } from "pino";
+import { ZodError, z } from "zod";
+
+import { envSchema } from "./env.js";
+
+const logger = pino({
+  level: "error",
+});
+
+export const configSchema = z.object({
+  appInsights: z.object({
+    connectionString: z.string().min(1),
+  }),
+});
+
+export type Config = z.TypeOf<typeof configSchema>;
+
+const configFromEnvironment = envSchema
+  .transform(
+    (env): Config => ({
+      appInsights: {
+        connectionString: env.APPLICATIONINSIGHTS_CONNECTION_STRING,
```

Comment:
since `APPLICATIONINSIGHTS_CONNECTION_STRING` was removed, it should also removed here